srv/salt/ceph/monitoring/prometheus/README.md

```diff
@@ -0,0 +1,74 @@
+# Monitoring
+By default DeepSea deploys a monitoring stack on the salt master. This consists
+of prometheus and Grafana, as well as the ceph_exporter of the salt master and
+the node_exporter on all salt minions.
+The prometheus configuration and scrape targets are setup automatically by
+DeepSea.
+
+## Alerting
+With version 0.8.6 DeepSea also deploys a list of
+[default alerts](files/ses_default_alerts.yml).
+
+### alertmanager configuration
+The alertmanager configuration file (also see [the upstream
+documentation](https://prometheus.io/docs/alerting/configuration/) sets, among
+other things, routes, receivers, inhibit rules and authentication for various
+notification channels (smtp, slack a.o.). Since these options are heavily
+dependent on a deployment, DeepSea doesn't ship any defaults here. For
```

Comment:
Since these options vary from deployment to deployment,
- or -
Since these options are very deployment-dependent,